code/game/objects/items/bedsheets.dm

```diff
@@ -18,6 +18,7 @@
 	w_class = WEIGHT_CLASS_TINY
 	dying_key = DYE_REGISTRY_BEDSHEET
 	resistance_flags = FLAMMABLE
+	light_system = MOVABLE_LIGHT
```

Comment:
This should not be applied to all bedsheets, it should instead only be added to `/obj/item/bedsheet/cosmos` and `/obj/item/bedsheet/double/cosmos` for the intended effect

Reply:
No other bedsheets have lights on them so it won't make an effect, if they somehow get lighting on them then they will use the movable light which is good

Reply:
From testing this is not the case, every bedsheet has a small light effect by default
![image](https://user-images.githubusercontent.com/81387903/219940346-d31ee393-a722-4de6-b285-8b395b22f091.png)

Reply:
Huh, didn't spot that.